bloggies_backend/src/models/email.ts

```diff
@@ -28,7 +28,8 @@ export default class Email {
                 subject = 'Regrets from Learning Circle';
```

Comment:
Would we need to send an email in the rejection case? or can we just have a page that displays on the front end once their status has changed to rejected?

Reply:
We will talk more out the flow here next week!